### DISCUSSION TITLE
Feature request: easier access to uBlock block/disable/allow features

### DISCUSSION THREAD

**Discussion by Mikaela** [dhowe/AdNauseam#2522] [Ideas]
I like disabling JavaScript everywhere by default for accessibility reasons, but I end up having to restore it on many sites. This requires first clicking the AdNauseam button on the toolbar and then the uBlock Origin logo where I can finally restore access to JavaScript.

I know it's just one extra click, but it feels like a lot more than using NoScript simultaneously especially on Firefox Android.

- https://github.com/dhowe/AdNauseam/discussions/2424

I wonder if AdNauseam could bring the JavaScript button and maybe the others somewhere easier to access with just one click?

**Comment by mneunomne** [comment]
@Mikaela we actually have a suggestion for a feature like this in this ticket https://github.com/dhowe/AdNauseam/issues/1109 , where a user can make the uBlock Menu the default one. Maybe it is indeed time to implement it. Thanks for the feedback!

**Comment by dhowe** [comment]
I think an advanced option in settings to foreground the ublock menu makes sense here, just need to figure out how it fits priority-wise

**Comment by mneunomne** [comment]
@Mikaela I just realized this can be achieved with AdNauseam already using the "MY RULES" functions, for example:
```
domain.com * 1p-script block
domain.com * 3p-script block
domain.com * inline-script block
```

this can be achived by using the advanced ublock popup menu also accessible from the AdNauseam menu:
<img width="680" alt="Screenshot 2025-03-17 at 16 01 35" src="https://github.com/user-attachments/assets/5d02dbcd-c8ce-4a20-83ed-af59205e0d19" />

**Reply by Mikaela** [reply]
How about on Android?

**Reply by Opening-Button-8988** [reply]
@mneunomne Thanks for this, I was having trouble finding this.

For anyone still unclear on this, click the Ublock Origin symbol at the bottom left of the extension menu (see image)

<img width="311" height="495" alt="image" src="https://github.com/user-attachments/assets/aa3f0cce-43e1-4330-94e9-02e0075dd095" />


Then click "More", shown at the bottom left.


<img width="247" height="404" alt="image" src="https://github.com/user-attachments/assets/b02aa725-edee-4a6a-bd22-aa904eb2bea8" />

Then it will show up. I had to apply at least one rule for the advanced menu to open by default. This should be explained in the Adnauseam wiki, if it does I couldn't find it when I searched "advanced".
